### DISCUSSION TITLE
Update: iPad Animation Course Release

### DISCUSSION THREAD

**Discussion by ota-works** [keiwando/toonsquid#415] [General]
Looks like the course is still being released, and very soon. Next week (Sept. 26th)

https://www.bloopanimation.com/ipad-animation/

I initially thought that due to the Procreate Dreams announcement, they might hold off on releasing the course and add Procreate Dreams' workflow to it, but I guess they're going ahead and releasing the course soon.
I just thought I'd share the info with everyone here.

**Comment by ota-works** [comment]
The iPad Animation course has been released and it looks AMAZING!!!

Here's one of the video from the course:

[https://www.youtube.com/watch?v=S_VS2iSGbhI](https://www.youtube.com/watch?v=S_VS2iSGbhI)

**Reply by keiwando** [reply]
That's great! Thanks for sharing the link

**Reply by ota-works** [reply]
Happy to help, Keiwan! I really hope your app keeps getting better and goes head-to-head with Industry professional animation softwares and apps

**Reply by rowby** [reply]
Like the link I posted earlier today — the link goes to a page not found.   Some bug with GitHub I assume

Please provide the name of the course so we can search YouTube for it.

Thanks
Rowby

**Reply by ota-works** [reply]
Hi, rowby!

Sorry, I'm now seeing your message. Search for "Bloop Animation iPad Animation course", and something helpful should pop up.

Cheers!
David

**Reply by keiwando** [reply]
I fixed the link in the original reply. When you write links with the special markdown link formatting, you need to put the link itself in parentheses and the text that should be visible and clickable in square brackets:
`[This is what the link will be visible as](https://example.com)`
will result in 
[This is what the link will be visible as](https://example.com)

You can also just paste the link without any formatting which works as well.

**Comment by rowby** [comment]
Ah thanks!


Rowby Goren
Shopify Expert and Partner
310-801-0110
https://www.Rowby.com
